helpers/README.md

```diff
@@ -0,0 +1,353 @@
+```
+[Back to top](#list-of-content)
+
+---
+### PhysicsConstraint
+```c
+typedef struct {
+	u8    lockRoot;    // Prevent physics rotating root limb
+	Vec2f rotStepCalc; // DEG, limits rot to next limb in main calc
+	Vec2f rotStepDraw; // DEG, limits rot on draw, smoothens output
+} PhysicsConstraint;
+```
+[Back to top](#list-of-content)
+
+---
+### PhysicsRigidity
+```c
+typedef struct {
+	s32   num;  // How many limbs will be smoothed with push
+	Vec3f push; // direction Z, pushes based on rot[0]
+	f32   mult; // How much the pushing fill affect
+	Vec3f rot;  // DEG, rigids towards, relative rot
+} PhysicsRigidity;
+```
+[Back to top](#list-of-content)
+
+---
+## Examples
+Here you set the arrays for physics to store/read values from. In this specific example the substruct is named `PonytailPhysics`, but you can name it whatever you like.
+
+```c
+
+typedef struct {
+	Actor         actor;
+	SkelAnime     skelAnime;
+	Vec3s         jointTable[SKEL_NPC_NUMBONES_DT];
+	Vec3s         morphTable[SKEL_NPC_NUMBONES_DT];
+	PhysicsStrand phyPonytail;
+	f32           phyJointLength[10]; // Number of limbs + 1
+	PhysicsJoint  phyJointTable[10];  // Number of limbs + 1
+	Vec3f         bodyPartsPos[6];
+	Vec3f         tailEnd;
+} EnNPC;
+```
+[Back to top](#list-of-content)
+
+---
+### PhysicsStrand
+Here are all the options that affect how the strand will be processed. These could also be stored in Actors struct, if these are set into it in init.
+```c
+PhysicsStrandInit sPonytailInit = {
+	.info = {
+		.numLimbs = 9,
+		.gravity = -2.0f,
+		.floorY = 2.0f,
+		.maxVel = 64.0f,
+		.velStep = 0.7f,
+		.velMult = 0.9f,
+	},
+	.gfx = {
+		.dlist = DL_PONYTAIL,
+		.scale = {
+			0.0055f,
+			0.0055f,
+			0.0055f
+		},
+		.segID = 0x0B,
+		.noDraw = true,
+	},
+	.spheres = {
+		.num = 6,
+		.centers = NULL,
+		.radius = 4.0f
+	},
+	.constraint = {
+		.lockRoot = false,
+		.rotStepCalc = {
+			.x = 45.0f,
+			.y = 45.0f,
+		},
+		.rotStepDraw = {
+			.y = 25.0f,
+			.x = 0.0f,
+		},
+	},
+	.rigidity = {
+		.num = 5,
+		.push = {
+			0,
+			0.2f,
+			-3.0f
+		},
+		.mult = 0.2f,
+		.rot = {
+			.x = 50.0f,
+			.y = 0.0f,
+			.z = 0.0f,
+		},
+	},
+	.limbsLength = {
+		-930.0f,
+		-1530.0f +  930.0f,
+		-2160.0f + 1530.0f,
+		-3150.0f + 2160.0f,
+		-4090.0f + 3150.0f,
+		-5190.0f + 4090.0f,
+		-6070.0f + 5190.0f,
+		-6810.0f + 6070.0f,
+		-400.0f,
+	},
+};
+```
+[Back to top](#list-of-content)
+
+---
+### ActorInit
+```c
+void EnNpc_Init(EnNpc* this, GlobalContext* globalCtx) {
+	Actor_SetScale(&this->actor, 0.0055f);
+	SkelAnime_InitFlex(globalCtx, &this->skelAnime, SKEL_NPC, ANIM_IDLE, this->jointTable, this->morphTable, SKEL_NPC_NUMBONES_DT);
+	
+	Physics_SetPhysicsStrand(&sPonytailInit, &this->phyPonytail, this->phyJointLength, this->bodyPartsPos);
+}
+```
+[Back to top](#list-of-content)
+
+---
+### OverridePostDraw
+In `SkelAnime` PostDraw you can get the "sphere" collision centers like so. This will prevent the limbs of the strand go through them.
```

Comment:
```suggestion
In `SkelAnime` PostDraw you can update the "sphere" collision centers like so (`bodyPartsPos`). This will prevent the limbs of the strand from going through the skeleton limbs.
```